### PR TITLE
BETA: Register midair.is-a.dev

### PR DESCRIPTION
**Pull request by zMidair**
Added `midair.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/midair.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "zMidair",
+        "email": "midair4546@gmail.com"
+    },
+    "record": {
+        "URL": "midair.is-a.dev"
```
Comment (CuteDog5695) [inline domains/midair.json:7]: invalid URL record, also you cant use your is-a.dev domain which you want in any of these records

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/midair.json (hunk 1 above)
Comment (zMidair) [reply domains/midair.json:7]: Wait, one sec, I will edit mb

Comment (zMidair) [reply domains/midair.json:7]: Can you edit the record value to “midair.ssite.link”? Idk how I can do that

Comment (zMidair) [reply domains/midair.json]: Updated

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/midair.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "zMidair",
+        "email": "midair4546@gmail.com"
+    },
+    "record": {
+        "URL": "midair.ssite.link"
```
[the remark this answers is not in the document]
Comment (zMidair) [reply domains/midair.json:7]: Alright, modifying

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Review by iostpa — CHANGES_REQUESTED** [review]
sorry, but rejecting it now. gives 404 page (on my PC i could see the page the second time but on my phone it isnt really letting me, so please fix the issue first)

**Comment by zMidair** [comment]
Hello, the 404 issue is now fixed.

**Comment by CuteDog5695** [comment]
> Hello, the 404 issue is now fixed.

alright

**Review by iostpa — APPROVED** [review]

**Pull request merged**